diff --git a/node_modules/react-native-reanimated/lib/module/common/logger.js b/node_modules/react-native-reanimated/lib/module/common/logger.js
index cb181b3..78440c4 100644
--- a/node_modules/react-native-reanimated/lib/module/common/logger.js
+++ b/node_modules/react-native-reanimated/lib/module/common/logger.js
@@ -66,6 +66,7 @@ function handleLog(level, message, options) {
 
   const config = getLoggerConfig();
   if (
+  message.includes('dependencies should only be used in web implementation') ||
   // Don't log if the log is marked as strict-only and the config doesn't
   // enable strict logging
   options.strict && !config.strict ||
diff --git a/node_modules/react-native-reanimated/src/common/logger.ts b/node_modules/react-native-reanimated/src/common/logger.ts
index 9dd8b46..049b052 100644
--- a/node_modules/react-native-reanimated/src/common/logger.ts
+++ b/node_modules/react-native-reanimated/src/common/logger.ts
@@ -93,6 +93,7 @@ function handleLog(
   'worklet';
   const config = getLoggerConfig();
   if (
+    message.includes('dependencies should only be used in web implementation') ||
     // Don't log if the log is marked as strict-only and the config doesn't
     // enable strict logging
     (options.strict && !config.strict) ||
